.github/workflows/performance-testing.yaml

```diff
@@ -74,11 +81,13 @@ jobs:
       - name: Start Docker Compose
         run: |
           JSONTAG=$(git show -s --format="%H")
-          docker pull ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG
-          docker image tag ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG stargateio/${{ matrix.docker-image }}:$JSONTAG
+          docker pull ${{ secrets.ECR_REPOSITORY }}/stargateio/jsonapi:$JSONTAG
```

Comment:
In copying steps from docker pull from this workflow for the new postman workflow, I realized that this workflow wasn't actually pulling the coordinator image from ECR. I updated both workflows with logic to pull the coordinator image according to the Stargate version referenced in the pom.